src/components/Swap/Summary/Details.tsx

```diff
@@ -57,6 +58,10 @@ function Detail({ label, value, color }: DetailProps) {
   )
 }
 
+const ToolTipBody = styled(ThemedText.Caption)`
+  max-width: 220px;
+`
+
```

Comment:
I don't think this needs to be dynamic as this size looks good on both small 300px and large widget widths, but am opening to changing if wanted